test/specs/collections/Form/Form-test.js

```diff
@@ -31,7 +32,9 @@ describe('Form', () => {
   common.hasUIClassName(Form)
   common.rendersChildren(Form)
 
-  common.implementsWidthProp(Form, _.without(SUI.SIZES, 'medium'), { propKey: 'widths' })
+  common.implementsWidthProp(Form, [], {
+    propKey: 'widths',
+  })
```

Comment:
Also fixed a test that produced propTypes' warning, `Form` can accept only `equal` width.